containers/geth/is-geth-synchronized.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/sh -e
+geth_address="$1"
+
+
+message="$(cat<<EOF
+{
+    "jsonrpc": "2.0",
+    "method":  "eth_syncing",
+    "params":  [],
```

Comment:
You could indent the `{ }` block for readablity. The extra whitespace should not break anything.